metrics/example/metrics.py

```diff
@@ -0,0 +1,63 @@
+import json
+import math,sys,time
+import random
+import requests
+import datetime
+
+print("*******************Custom Metrics Generator***************************")
+url = "http://dkube-exporter.dkube:9401/modelmonitor/metrics"
+
+monitor_id = sys.argv[1]
+print("monitor id is ",monitor_id)
+
+metric_value = random.random()
+
+payload = json.dumps({
+  "id": monitor_id,
+  "timestamp": math.ceil(datetime.datetime.utcnow().timestamp()),
```

Comment:
config.json should be mounted here. Can you provide a function to calculate the end of last sample based on run_frequency. You can use metrics.py in monitoring repo for reference.

Reply:
@gaurav-oc make sure config.json is mounted for custom image

Reply:
inside custom container ?

Reply:
its mounted same as we mount for other workflow container